tools/rum-models-generator/Sources/RUMModelsGeneratorCore/Input/ObjcInterop/ObjcInteropType.swift

```diff
@@ -0,0 +1,135 @@
+/*
+* Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+* This product includes software developed at Datadog (https://www.datadoghq.com/).
+* Copyright 2019-2020 Datadog, Inc.
+*/
+
+import Foundation
+
+/// Type-safe Swift ↔ Objc interopability schema.
+internal protocol ObjcInteropType: AnyObject {}
+
+/// Any `@objc class` which manages Swift `struct`.
+internal protocol ObjcInteropClass: ObjcInteropType {
+    var managedSwiftStruct: SwiftStruct { get }
```

Comment:
A bit of a nitpick, but would `wrappedSwiftStruct` or `bridgedSwiftStruct` describe better the relationship? What would the management involve otherwise?
The same questions will apply for any `wrappedSwift*` in that file.

Reply:
It used to be `wrappedSwiftStruct` in my draft implementation, but I changed to `managedSwiftStruct` as it didn't "sound good" to me. The `bridgedSwiftStruct` however sounds great 👍 - changed.